src/env.cpp

```diff
@@ -98,6 +98,36 @@ int applyUint32Setting(int (*f)(MDB_env *, T), MDB_env* e, Local<Object> options
     return rc;
 }
 
+class SyncWorker : public Nan::AsyncWorker {
+  public:
+    SyncWorker(MDB_env* env, Nan::Callback *callback)
+      : Nan::AsyncWorker(callback), env(env) {}
+
+/*    ~SyncWorker() {
+      delete env;
+    }*/
```

Comment:
You deleted the same lines in your previous commit, I assume these are here by mistake?

Reply:
I think I deleted it in the later/2nd commit. It makes sense that isn't needed, right?